src/app/shared/filesystem/file-list-view/file-list-view.component.scss

```diff
@@ -33,12 +33,12 @@
 
 .grid-row {
   display: grid;
-  grid-template-columns: 2fr 2fr 1fr 25px;
+  grid-template-columns: 2fr 2fr 2fr 1fr 25px;
   grid-column-gap: 20px;
   align-items: center;
   width: 100%;
   background: #fff;
-  padding: 0 8px 8px 0;
+  padding: 0 0 8px 0;
```

Comment:
Why is the right-padding removed here?

Reply:
If I didn't, we'd have this: 
![sad](https://user-images.githubusercontent.com/37353360/48012532-3291f800-e0f0-11e8-8516-2563af567d22.png)

Instead of this:
![happy](https://user-images.githubusercontent.com/37353360/48012544-3b82c980-e0f0-11e8-9a23-5c16916848b0.png)